src/app/member/components/MemberList.tsx

```diff
@@ -199,12 +85,21 @@ export const MemberList = () => {
   ];
 
   const table = useReactTable({
-    data: members,
+    data,
     columns,
     getCoreRowModel: getCoreRowModel(),
   });
 
-  const groupedData = useGroupedData(members, "shift");
+  useEffect(() => {
+if (!success) {
+    toast({
+      title: "No Members Found",
```

Comment:
We should handle it more like UI state. We don't need to rely on API. When you're fetching data, the component won't render, instead loading.tsx would be in process. You can place two  conditions here:
- If data, show table.
- else show Standardpage. @muhammad-bilal-ali-khokhar let's create a task to create and incorporate standard page.

Reply:
Sir @Monfernape  , I have created `ticket` for standard page